lms/templates/api/canvas/oauth2_redirect_error.html.jinja2

```diff
@@ -0,0 +1,38 @@
+{% extends "lms:templates/base.html.jinja2" %}
+
+{% block content %}
+  <div class="Dialog__background" style="z-index: 1;"></div>
```

Comment:
Hmm ... there is a hazard here that it might not be obvious to someone modifying `Dialog.js` and `_Dialog.scss` that styles are referenced outside of those two files. Re-using classes between server and client-rendered HTML is something we'll want to do though.

For the moment, I think it would help simply to update the documentation [here](https://github.com/hypothesis/lms/blob/e740307635ad79d5ae725d6c0c7a18ef7dfb8ddc/lms/static/styles/file-picker-app.scss#L13) to indicate that these SASS components are used by the frontend and server-rendered HTML.

Reply:
Done